src/java.desktop/windows/classes/sun/awt/shell/Win32ShellFolderManager2.java

```diff
@@ -167,9 +167,6 @@ static Win32ShellFolder2 getDesktop() {
         if (desktop == null) {
```

Comment:
`import java.util.stream.Stream;` unused

Reply:
Doesn't seem related to this change. I'll fix it here since a simple build should prove it is safe, but refactoring and non-SM clean up is not a goal in these fixes.

Reply:
In this case, it is related because you have removed its use:

`- private static File[] checkFiles(Stream<File> filesStream, @SuppressWarnings("removal") SecurityManager sm) {`